pkg/providers/vsphere/vsphere.go

```diff
@@ -274,7 +274,17 @@ func (p *vsphereProvider) PostMoveManagementToBootstrap(_ context.Context, _ *ty
 	return nil
 }
 
+// TODO: Remove this field for v0.17.x, adding a warning as of v0.16.0
+func warnIfCSIEnabled(disableCSI bool) {
+	if !disableCSI {
+		// Need to add spacing to make the log message look neat
+		logger.MarkWarning("  Warning: Installing CSI through EKS Anywhere is deprecated. Refer to the official documentation for more details on " +
```

Comment:
I think that wording is fine. Saying that something "is deprecated" means that it will be going away in the future, but has not gone away yet.

Reply:
@vivek-koppuru Do we have a ticket users can track? Its useful to include that in warnings. No worries if not.

Reply:
We have this issue https://github.com/aws/eks-anywhere/issues/5517 but if we feel that is not the right one, we can create a different one. I am still leaning towards putting it in our documentation and changelog and not here because I don't want to make the message bigger, unless we think that is not a problem.